### ISSUE TITLE
🛠 feat: Color skin tone filter

### ISSUE THREAD

**Issue by FelipeFTN** [FelipeFTN/Emoji-Copy#42]
# Skin tone selector is not working as expected.
Maybe building a new SQLite query for this function should work, then it will only query the emojis with the right skin tone or without any skin-tone (like objects perhaps).

**Comment by yob** [comment]
This would be super helpful. I recently switched from the abandoned gnome-emoji-selector and this is one of the few behaviour changes that feels a bit surprising - it takes more keypresses to select an emoji than previously:

![Screenshot from 2024-08-05 12-08-56](https://github.com/user-attachments/assets/d6b57722-7fdf-4f92-b978-3a5050405c19)

**Comment by FelipeFTN** [comment]
Yeah, @yob, this is a bit of a complicated feature hahaha but It's still possible. I might start working on this next week! :rocket:

**Comment by mawkler** [comment]
I noticed that there's a color selector under the "faces" section, but it doesn't seem to do anything on my machine. The emoji are still the default yellow skin color after selecting them. Am I doing something wrong?

![Image](https://github.com/user-attachments/assets/a5e74702-bb76-4476-ac22-c9c19364d686)

I'm on version 26.

**Comment by jessicarod7** [comment]
On version 28, still present. @mawkler I think there are two separate issues:

1. The skin tone and gender selectors work when searching for an emoji, but they don't work when scrolling through the selector panel.

![Image](https://github.com/user-attachments/assets/bd75b887-af0b-41fd-a172-4704a0d4140f)

2. There's a categorization mistake. The first panel is listed as "Smiles & Body", but the body parts (hands, legs, etc.) are mistakenly included in the next section, "Peoples & Clothing".

![Image](https://github.com/user-attachments/assets/7541a8a6-6299-48e0-8a83-7768e4ca60b7) ![Image](https://github.com/user-attachments/assets/146e6118-4f6d-4aeb-88fb-104f6f9a9064)

**Comment by FelipeFTN** [comment]
@elliotwutingfeng It would require a new release just to omit this functionality. So it might be better to spend some time trying to actually implement the feature by the time we start working on this! 💯

**Comment by FelipeFTN** [comment]
Hey folks! Just opened a Pull Request showcasing the feature! Feel free to test the branch by checking out into it and running `make debug` a couple times for the Emoji Copy extension properly refresh in the emulated wayland environment!

**Please take a look and test this the feature in your PCs as well! 🙏** 

The Pull Request is open for suggestions and comments! 💯
